### ISSUE TITLE
Creating custom test report

### ISSUE THREAD

**Issue by benbcai** [cerner/terra-toolkit#370]
# Feature Request

## Description
When PR #369 bumped to axe-core 3.4.0 it filtered any test failures caused by the set of new accessibility rules for passivity. We should look into creating a custom test report that prints out warnings for these accessibility failure at the end of the test execution. Displaying these warnings will provide higher visibility to teams so they can become aware and make plans for correcting the failures when these new rules become enabled in the future.

The custom test report could potentially be used to display other types of testing information other than just warnings.

**Comment by emilyrohrbough** [comment]
similar to: https://github.com/cerner/terra-toolkit/issues/173

copied from that issue:

I would be interested creating a custom reporter that will record start and end time of tests for the form factor and local tested and on what selenium host/session to make debugging infrastructure failures easier.
